Customers app

Abdullah-Waqas13341 opened the pull request:
Made the Customers App Payment Method is not Fully Complete Working on it

Commit:
Schema Implementation

Commit:
Forms

Commit:
Almost Completed Customers App Stripe Left

rabbiasajjad-devsinc commented on customers/templates/customers/services_list.html:
It is recommended to put all the styling in separate CSS files.

rabbiasajjad-devsinc commented on customers/templates/customers/payment.html:
Put the Stripe API key in .env file

rabbiasajjad-devsinc commented on core/views.py:
Remove redundant imports

rabbiasajjad-devsinc commented on core/views.py:
Put MAX_ATTEMPTS and LOCKOUT_TIME in .env file

Commit:
Payment Method Implemented

Commit:
Payment Using Stripe is Complete
Api Keys,Max Attempts and Lockout Time is moved to .env
.env was moved to git ignore
All the internal css is moved to static/app/css/style.css
